src/index.js

```diff
@@ -28,14 +28,24 @@ import 'styles/main.scss';
       const $selection = $this.find('p.iqdropdown-selection').last();
       const $menu = $this.find('div.iqdropdown-menu');
       const $items = $menu.find('div.iqdropdown-menu-option');
-      const settings = $.extend(true, {}, defaults, options);
       const itemCount = {};
       let totalItems = 0;
 
+      const settings = $.extend(
+        true,
+        {
+          getCustomMessage: () => {
```

Comment:
`setSelectionText` would be a better name for this function since it would override the `selectionText` option

Reply:
rename function to setSelectionText